src/util/delay.test.js

```diff
@@ -11,14 +11,8 @@ describe('delay', () => {
     })
 
     test('should return a promise which resolves', () => {
-        expect.assertions(2)
-        let assertionFunc = jest.fn(() => {
-            return expect(assertionFunc).toBeCalled()
-        })
-        delay(10).then(() => {
-            assertionFunc()
-        })
-        expect(assertionFunc).not.toHaveBeenCalled()
+        expect.assertions(1)
+        expect(delay(10)).resolves.toBe()
```

Comment:
This doesn't work as wanted. resolves returns a promise which is ignored right now. See https://facebook.github.io/jest/docs/en/expect.html#resolves.
So just use the previous version of the test and move `expect(assertionFunc).toBeCalled()` to the end of the test. That should do it if I don't miss anything.